run-basic-batch-operators-test.sh

```diff
@@ -62,22 +83,20 @@ cleanup() {
 	oc delete mutatingwebhookconfigurations controller.devfile.io || true
 	oc delete validatingwebhookconfigurations controller.devfile.io || true
 
-	# cleanup any leftovers
-	# https://docs.openshift.com/container-platform/4.14/operators/admin/olm-deleting-operators-from-cluster.html
-	oc get csv -n openshift-operators | grep -v packageserver | grep -v NAME | awk '{print " oc delete --wait=true csv " $2 " -n openshift-operators"}' | bash || true
-	oc get csv -A | grep -v packageserver | grep -v NAME | awk '{print " oc delete --wait=true csv " $2 " -n " $1}' | bash || true
-	oc get subscriptions -A | grep -v NAME | awk '{print " oc delete --wait=true subscription " $2 " -n " $1}' | bash || true
-	oc get job,configmap -n openshift-marketplace | grep -v NAME | grep -v "configmap/kube-root-ca.crt" | grep -v "configmap/marketplace-operator-lock" | grep -v "configmap/marketplace-trusted-ca" | grep -v "configmap/openshift-service-ca.crt" | awk '{print " oc delete --wait=true " $1 " -n openshift-marketplace" }' | bash || true
+	# Leftovers specific to certain operators
+	oc delete ValidatingWebhookConfiguration sriov-operator-webhook-config || true
+	oc delete MutatingWebhookConfiguration sriov-operator-webhook-config || true
 }
 
 waitDeleteNamespace() {
```

Comment:
As far as I know, we use snake_case for shell scripting, not camelCase. You can find more information [here](https://dev.to/ankitmalikg/difference-among-casings-snake-case-vs-camel-case-vs-pascal-case-vs-kebab-case-2m8b).

Reply:
I know that usually snake_case is used but I prefer to keep it camel case

Reply:
@edcdavid @sebrandon1 I think I saw snake_case in other scripts. It should be consistent everywhere.

Reply:
Fixed